builder/builder-next/worker/worker.go

```diff
@@ -212,7 +213,7 @@ func (w *Worker) Prune(ctx context.Context, ch chan client.UsageInfo, info ...cl
 // Exporter returns exporter by name
 func (w *Worker) Exporter(name string, sm *session.Manager) (exporter.Exporter, error) {
 	switch name {
-	case "moby":
+	case mobyexporter.Moby:
 		return w.Opt.Exporter, nil
```

Comment:
Not for this PR; wondering if perhaps Workers should haven an option to get passed a list of exporters when creating them, and keep a `map[string]Exporter` 🤔, or `map[string]ExporterConstructor()`